jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,476 @@
+`/project/{project-slug}/pipeline/mine` | This endpoint enables you to return all pipelines for the project that you have triggered by making a GET request and passing the `project-slug` parameter in the request..
+`/project/{project-slug}/envvar` | This endpoint enables you to list all current environment variables for a specific project by making a GET request and passing the `project-slug` parameter in the request.
+`/project/{project-slug}/envvar` | This endpoint enables you to create an environment variable for a project by making a POST request and passing the `project-slug` parameter in the request.
+`/project/{project-slug}/envvar/{name}` | This endpoint enables you to delete an existing environment variable from a project by making a DELETE request and passing the `project-slug` and `name` parameters in the request.
+`/project/{project-slug}/envvar/{name}` | This endpoint enables you to retrieve a masked value for an environment variable for a project by making a GET request and passing the `project-slug` and `name` parameters in the request.
+
+### Job Endpoints
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`/project/{project-slug}/job/{job-number}` | This endpoint enables you to retrieve job details by making a GET request and passing the project-slug and job-number parameters in the request.
+`/project/{project-slug}/job/{job-number}/cancel` | This endoint enables users to cancel a job by making a POST request and passing the `project-slug` and `job-number` parameters in the request.
+`/project/{project-slug}/{job-number}/artifacts`  | This endpoint enables users to retrieve artifacts for a job by making a GET request and passing the project-slug and job-number parameters in the request.
+`/project/{project-slug}/{job-number}/tests` | This endpoint enables you to retrieve test metadata for a job by making a GET request and passing the `project-slug` and `job-number` parameters in the request.
+
+### Workflow Endpoints
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`/workflow/{id}` | This endpoint enables users to you to return a workflow by making a GET request and passing the `id` parameter in the request.
+`/workflow/{id}/cancel` | This endoint enables you to cancel a workflow by making a POST request and passing the `id` parameter in the request.
+`/workflow/{id}/job`  | This endpoint enables you to return the jobs associated with a workflow by making a GET request and passing the `id` parameter in the request.
+`/workflow/{id}/rerun` | This endpoint enables you to rerun a workflow by making a POST request and passing the `id` parameter in the request.
+
+## API Syntax
```

Comment:
Not sure this section is actually interesting to the user.

Reply:
commented out on page.